### ISSUE TITLE
Finding an org for this package

### ISSUE THREAD

**Issue by meggart** [JuliaIO/DiskArrays.jl#3]
As already discussed in #2 before registering this package I would like to decide about the org where the repo should be hosted. It was suggested that JuliaArrays would be a good fit, it is not obvious who to contact so I am pinging @mbauman who was involved in the initial discourse discussion, if he can comment if this package would meet the criteria to be included in JuliaArrays.

**Comment by rafaqz** [comment]
Why not JuliaIO ? Seems like a good place for it.

**Comment by meggart** [comment]
Given that the package has some dependents by now it might be a good time to revive this discussion. @mkitti I would be interested in your opinion on potentially moving the package to JuliaIO.

**Comment by mkitti** [comment]
That seems like a good idea to me. Are you a JuliaIO member yet?

**Comment by meggart** [comment]
Yes, I am already member of JuliaIO, but didn't just want to transfer the project without asking.

**Comment by rafaqz** [comment]
I'm not a JuliaIO member, so you may need to make me a package maintainer again after moving the repo or add me as a org member.

**Comment by mkitti** [comment]
@rafaqz Check your email or inbox for an invite to JuliaIO.

**Comment by asinghvi17** [comment]
Is there still interest in moving to JuliaIO?

**Comment by meggart** [comment]
Ah yes of course. I guess I only need to hit the button, or is there anything else to consider. I just don't want to break things...

**Comment by asinghvi17** [comment]
The button and changing where the docs / CI / etc point to, but I can handle that in a quick PR :)

**Comment by rafaqz** [comment]
Yeah its surprisingly painless, github keeps all the redirects so most things keep working. Just some badges break

**Comment by meggart** [comment]
Ok, done. We are on JuliaIO now 🥳 Let's see what breaks and change the urls then.

**Comment by rafaqz** [comment]
We can make a PR to The registry to change the url officially too

**Comment by meggart** [comment]
> We can make a PR to the registry to change the url officially too

Ok Registry PR is here: https://github.com/JuliaRegistries/General/pull/119065

**Comment by asinghvi17** [comment]
General URL is now changed.

**Comment by meggart** [comment]
Good, it looks like the only thing that is still broken is the codecov link. Not sure what must be changed to make this work again...

**Comment by rafaqz** [comment]
Closing, we have an org now :D
